### PR TITLE
feat: new contributor highlight card

### LINKED ISSUE #1176: Feature: Highlights Design System Improvements
### Type of feature

🍕 Feature

### Current behavior

During a Design Exploration we did - [[Figma](https://www.figma.com/file/KwtbAJzP2Y8zENhVsKplRo/OpenSauced?type=design&node-id=0%3A1&t=hCipvyDYRn54dz4z-1)]

1. New types of Highlights will need a type added 
2. Repo tag should be added to the highlight [isse, pr, and blog post]
3. Milestone highlights

## New design
- type on the top left
- Repo tag bottom right
- Quick share to twitter icon top right

Screen Shot 2023-05-13 at 1 15 52 AM

## Milestones
- one with post (PR)
- one without post

Screen Shot 2023-05-13 at 1 15 32 AM

### Additional context

We will need to update the logic for highlights to include the check for type of highlight. For now treat everything as a pull request highlight when shipping live.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [ ] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by OgDev-01**
## Description
This PR updates the contributor's highlight card UI to now support highlights for 

This PR adds highlights for
 - Pull requests
 - Issues 
 - Blogs to the contributor's highlight card UI.

For the time being, failing to provide a type will result in a Pull request. 

This PR  also includes a story for the contributor's highlights card.

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents
Closes #1176 

### default
Screenshot 2023-07-27 at 14 56 52

### Issue
Screenshot 2023-07-27 at 14 58 12

### Blog
Screenshot 2023-07-27 at 14 58 49

### New looks in feeds
Screenshot 2023-07-27 at 14 59 33

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [x] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Review by jpmcb — COMMENTED** [review]
I'm abit confused on this change: should I be able to add issues or blog posts when creating a highlight?

When I an issue URL, it fails since it's not a valid PR link:

Screenshot 2023-07-27 at 9 48 43 AM

This is also abit confusing from a user experience standpoint since the copy in the URL field is: "Paste your PR URL and get it auto-summarized!" Are PRs only accepted? Is that field only for auto-summarizing PRs?

**Comment by bdougie** [comment]
in consistency with titles and without. Would love some spacing when there is a title. 

## without title
Screen Shot 2023-07-27 at 9 53 55 AM

## with title
Screen Shot 2023-07-27 at 9 53 51 AM

**Review by brandonroberts — CHANGES_REQUESTED** [review]

File: components/molecules/ContributorHighlight/contributor-highlight-card.tsx [hunk 1]
```diff
@@ -231,24 +286,26 @@ const ContributorHighlightCard = ({
...
-                    className="flex gap-2.5 py-1 items-center pl-3 pr-7"
+                    className="flex items-center p-3 transition rounded-full hover:bg-light-orange-5"
                   >
-                    <FiTwitter size={22} />
-                    <span>Share to Twitter</span>
+                    <BsTwitter className="text-lg text-light-orange-9 md:text-xl" />
                   </a>
-                </DropdownMenuItem>
+                </Tooltip>
+                <DropdownMenuTrigger className=" py-2 px-2 rounded-full data-[state=open]:bg-light-slate-7">
```
Comment (NsdHSO) [inline components/molecules/ContributorHighlight/contributor-highlight-card.tsx:303]:
```suggestion
                <DropdownMenuTrigger className="py-2 px-2 rounded-full data-[state=open]:bg-light-slate-7">
```

**Review by brandonroberts — APPROVED** [review]

**Review by babblebey — COMMENTED** [review]
🍕

**Pull request merged**